Gemfile

```diff
@@ -132,7 +132,7 @@ end
 
 group :vmware, :manageiq_default do
   manageiq_plugin "manageiq-providers-vmware"
-  gem "vmware_web_service",             "~>0.1.4"
+  gem "vmware_web_service",             "~>0.2.0"
```

Comment:
Unfortunately https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_vim_broker_worker.rb and https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_ems_refresh_core_worker/runner.rb still need this.